output/qaqc_historic.sql

```diff
@@ -0,0 +1,76 @@
+--
+-- PostgreSQL database dump
+--
+
+-- Dumped from database version 13.4 (Debian 13.4-4.pgdg110+1)
+-- Dumped by pg_dump version 14.0
+
+SET statement_timeout = 0;
+SET lock_timeout = 0;
+SET idle_in_transaction_session_timeout = 0;
+SET client_encoding = 'UTF8';
+SET standard_conforming_strings = on;
+SELECT pg_catalog.set_config('search_path', '', false);
+SET check_function_bodies = false;
+SET xmloption = content;
+SET client_min_messages = warning;
+SET row_security = off;
+
+SET default_tablespace = '';
+
+SET default_table_access_method = heap;
+
+--
+-- Name: qaqc_historic; Type: TABLE; Schema: public; Owner: postgres
+--
+
+CREATE TABLE public.qaqc_historic (
+    version text,
+    b_likely_occ_desc integer,
+    b_large_alt_reduction integer,
+    b_nonres_with_units integer,
+    units_co_prop_mismatch integer,
+    partially_complete integer,
+    units_init_null integer,
+    units_prop_null integer,
+    units_res_accessory integer,
+    outlier_demo_20plus integer,
+    outlier_nb_500plus integer,
+    outlier_top_alt_increase integer,
```

Comment:
maybe its worth keeping this column for consistency, but just to note this means "flag the top twenty alterations by unit increase", so the number of things flagged will always be 20, making this kind of meaningless

Reply:
interesting, good catch. I think we should keep it in and filter it on the app side for simplicity's sake